custom-connectors/PUGGamifiedEngagement/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "PICNIC CONNECTOR",
+    "description": "PICNIC CONNECTOR",
+    "contact" : {
+      "name" : "Pug Interactive",
+      "url" : "https://www.puginteractive.com",
+      "email" : "support@puginteractive.com"
+    }
+  },
+  "host": "connector.picnicplatform.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/players/create_player": {
+      "post": {
+        "summary": "Create Player",
+        "description": "Create Player",
+        "operationId": "Create_Player",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": " application/vnd.sno-ge.iapi+json",
+            "description": "Content-Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": " application/vnd.sno-ge.iapi+json",
+            "description": "Accept",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "player": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "profile": {
+                      "type": "object",
+                      "properties": {
+                        "first_name": {
+                          "type": "string",
+                          "description": "first_name"
+                        },
+                        "last_name": {
+                          "type": "string",
+                          "description": "last_name"
+                        }
+                      },
+                      "description": "profile"
+                    },
+                    "external_realm": {
+                      "type": "string",
+                      "description": "external_realm"
+                    },
+                    "external_realm_id": {
+                      "type": "string",
+                      "description": "external_realm_id"
+                    }
+                  },
+                  "description": "player"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@smart-jp It is recommended that to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.